src/Banner.js

```diff
@@ -0,0 +1,37 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/core'
+import styled from '@emotion/styled'
+import F from 'futil'
+import { defaultProps } from 'recompose'
+import { Icon, Text, Flex } from '.'
+import theme from './theme'
+
+let getColor = F.aliasIn(theme.colors)
+
+let bannerStyle = (bgColor, textColor) => ({
+  backgroundColor: getColor(bgColor),
+  '*': { color: `${getColor(textColor)} !important` },
+})
+
+let BaseBanner = ({ children, icon = 'priority_high', ...props }) => (
+  <Flex
+    alignItems="center"
+    justifyContent="center"
+    css={{ padding: theme.spaces[1] }}
+    {...props}
+  >
+    <Icon icon={icon} style={{ marginRight: theme.spaces[1], verticalAlign: 'text-bottom' }} />
+    <Text small as='span'>{children}</Text>
+  </Flex>
+)
+
+let Banner = styled(BaseBanner)(bannerStyle('primaries.0', 'neutrals.0'))
```

Comment:
We should define these pairs as part of the theme - text and background colors likely apply in other contexts, too

Reply:
that's a neat idea 🙂

I'm honestly not sure if we use the same combinations consistently enough for it to be useful - it should be easier to tell after a few more components are done, at which point we'll most likely do a refactoring pass anyway

Reply:
if we aren't defining these first class, we should make it an object like we did with Buttons